### ISSUE TITLE
Add configurable orchestration

### ISSUE THREAD

**Issue by arr28** [DodgyJammers/jammypiece#15]
i.e. send "program change" event(s) for the harmonised part(s).

**Comment by rgefryer** [comment]
Maybe.  I think we'd be best served focussing on a single accompaniment style/orchestration initially, and only branching out if we get it totally sorted.  This always needs to be user selectable in any case - sensible choice of orchestration depends a lot on the patch used for the solo line, and there's no way that the code can understand that.

**Comment by arr28** [comment]
Yes - I was certainly thinking user-configurable rather than "learned".
